cmd/snap/cmd_quota.go

```diff
@@ -178,7 +193,7 @@ func (x *cmdSetQuota) Execute(args []string) (err error) {
 		// currently support that, so currently all snaps specified here are
 		// just added to the group
 
-		chgID, err = x.client.EnsureQuota(x.Positional.GroupName, x.Parent, names, 0)
+		chgID, err = x.client.EnsureQuota(x.Positional.GroupName, x.Parent, names, nil)
```

Comment:
Related to my first question, it seems we do need to handle nil with postQuotaData.